app/models/order.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Order < ApplicationRecord
+  enum status: { pending_shipment: 0, completed: 1 }
```

Comment:
I think enum is easier to understand as string than integer, but what do you think?